Documentation/ci-config/config-plugins-audit.sh

```diff
@@ -1,16 +1,30 @@
 #!/usr/bin/env bash
 
 ##
-# @file config-plugins-audit.sh
-# @brief add plugin modules for a github repository
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
Can we ignore the original Apache2 here? (you are the author, so you should be able to drop Apache2)